### ISSUE TITLE
Duplicate keys on passing array of ReactElement

### ISSUE THREAD

**Issue by pbshgthm** [appfigures/react-easy-emoji#18]
I have defined a custom component that adds emoji

```typescript
export function RenderEmoji({ content }: { content: any }) {
  return emoji(content, (code, string, key) => (
    <img srrc={`https://twemoji.maxcdn.com/2/svg/${code}.svg`} alt={string} key={key} />
  ))
}
```

When content is a ReactElement array, it sometimes throws the following warning
`Encountered two children with the same key`. This happens as the key is offset from the start of that particular element so when content is `["🧡 abc", "🧡 xyz"]` both the returned images have the same key ` 0 `. 

How to handle this situation?

**Comment by oztune** [comment]
Great catch! We'll look into this.

**Comment by oztune** [comment]
This is now fixed in v1.6.1. Thanks for letting us know!

**Comment by pbshgthm** [comment]
Thanks for the super fast fix!
